modules/afsmtp/afsmtp.c

```diff
@@ -626,6 +666,8 @@ afsmtp_dd_init(LogPipe *s)
     return FALSE;
 
   g_list_foreach(self->headers, (GFunc)afsmtp_dd_init_header, cfg);
+  g_list_foreach(self->rcpt_tos, (GFunc)afsmtp_dd_init_recipient, cfg);
+
```

Comment:
This new function added "afsmtp_dd_init_recipient" is an essential step in the template compiling, 
it works like the afsmtp_dd_init_header method(one line above it) , it compiles(changes the $macro parts) the recipient's address strings into their addr_tmpl.